_raml/apps/products/examples/get_product_productid_response.json

```diff
@@ -88,6 +88,16 @@
   ],
   "stocklevel": 100,
   "minStocklevel": 10,
+  "videos": [
+    {
+      "type": "YouTube",
+      "source": "08154713"
```

Comment:
Not possible because were using not the complete url for YouTube. I can use the YouTube video id here. For other video providers the url or  an other identifier can be used if correctly implemented in the ui